app/javascript/styles/components.scss

```diff
@@ -2168,6 +2168,9 @@ button.icon-button.active i.fa-retweet {
   height: 100%;
   display: flex;
   height: 48px;
+  @media screen and (max-width: 1024px) {
+    margin-right: 10px;
```

Comment:
This margin would needed even for PC if column setting doesn't exists. How about removing left padding from `.column-header__button` instead of media queries on here?

Reply:
Thank you for your advice! I will try that.